core/src/main/java/io/questdb/griffin/engine/QueryProgress.java

```diff
@@ -120,23 +134,30 @@ public static void logEnd(
                 log.I$();
             }
         }
+        if (queryTrace != null && engine.getConfiguration().isQueryTracingEnabled()) {
+            queryTrace.isJit = isJit;
+            queryTrace.timestamp = config.getMicrosecondClock().getTicks();
+            queryTrace.queryText = sqlText;
+            queryTrace.executionNanos = TimeUnit.NANOSECONDS.toMicros(durationNanos);
```

Comment:
nit: that is a verbose way to divide by 1000 :)

Reply:
Resolved in 6fc0c8de55533cf196e92bca94540d880c220508